src/usr/share/netcheck/netcheck.py

```diff
@@ -45,12 +45,15 @@ class NetCheck(object):
     actually on the Internet.
     """
 
-    def __init__(self, config):
+    def __init__(self, config, broadcaster):
         """Constructor.
 
         config: The program configuration dictionary.
+        broadcaster: An already configured parkbenchcommon.broadcaster.Broadcaster object.
```

Comment:
Since no configuration happens other than initialization, just say what type of object it is.